src/schema/users.ts

```diff
@@ -2116,7 +2125,7 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         amount: recoverCost,
       };
 
-      const currentStreak = oldStreakLength + streak.current;
+      const currentStreak = oldStreakLength + streak.current!;
```

Comment:
@sshanzel for you sanity check, here `streak.current` could be `undefined`, I unwrapped it with `!` operator, but we might want to return early if `streak.current` does not exist? Maybe cause of some bugs?

Reply:
Should always exist. When a user is created, we trigger a user streak insert then the default value for this property is 0.

Reply:
I made `current` not optional in, can you check if ok @sshanzel? 1191e9cbf75a52a1ea21c45791987350daa57e6a

Reply:
Yes. Having it required is the way to go 🙌